### PR TITLE
Fix layout with search

### PR DESCRIPTION
**Pull request by octobrekids**
In this branch, 
- fix History Page Layout
- fix Task Page Layout 
- fix Dashboard Page Layout
- add dependency react-test-renderer and @types/react-test-renderer
- fix LayoutWithSearch Component (with isPadding props)
- add LayoutWithSearch Unit Test 
- add LayoutWithSearch storybook
- fix dashboard wrapper padding

### CODE REVIEW

File: src/common/components/Layouts/LayoutWithSearch/index.tsx [hunk 1]
```diff
@@ -104,17 +104,25 @@ const HeaderWrapper: React.FC = () => {
...
-          <Content style={{ overflow: 'initial' }}>
-            <h1>{title}</h1>
-            <hr />
-            {children}
-          </Content>
+          {router.pathname === '/dashboard' ? (
+            <Content style={{ padding: '5% 0% 5% 0%', marginLeft: '5%' }}>
+              {children}
+            </Content>
+          ) : (
```
Comment (yee2542) [inline src/common/components/Layouts/LayoutWithSearch/index.tsx:119]: magic logic

**Review by yee2542 — CHANGES_REQUESTED** [review]

File: src/common/components/Layouts/LayoutWithSearch/index.tsx (hunk 1 above)
Comment (yee2542) [reply src/common/components/Layouts/LayoutWithSearch/index.tsx:119]: maybe just add some props to allow enable/disable padding

Comment (octobrekids) [reply src/common/components/Layouts/LayoutWithSearch/index.tsx]: solve at 9e24d2430cead55c9c61f2ad9dc23aa1955a053d

File: src/common/components/Layouts/LayoutWithSearch/__snapshots__/index.spec.tsx.snap [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Component/LayoutWithSearch test Should match snapshots 1`] = `ShallowWrapper {}`;
```
Comment (yee2542) [inline src/common/components/Layouts/LayoutWithSearch/__snapshots__/index.spec.tsx.snap:3]: it seem not capture a real rendering a component, please check

File: src/common/components/Layouts/LayoutWithSearch/index.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import LayoutWithSearch from '.';
+import { ContentWithPadding } from './styled';
+
+describe('Component/LayoutWithSearch test', () => {
+  it('Should match snapshots', () => {
+    const wrap = shallow(<LayoutWithSearch />);
+    expect(wrap).toMatchSnapshot();
```
Comment (yee2542) [inline src/common/components/Layouts/LayoutWithSearch/index.spec.tsx:9]: render it with your new lib would be solved that I mentioned before

**Review by yee2542 — CHANGES_REQUESTED** [review]
snapshot look it will not garuntee your code and check with clear the compiler warning.

File: src/common/components/Layouts/LayoutWithSearch/index.spec.tsx (hunk 3 above)
Comment (octobrekids) [reply src/common/components/Layouts/LayoutWithSearch/index.spec.tsx]: fixed at f4a6fb4f1c5a3f92abc8256fafb9511c28d0eab8

File: src/common/components/Layouts/LayoutWithSearch/__snapshots__/index.spec.tsx.snap (hunk 2 above)
Comment (octobrekids) [reply src/common/components/Layouts/LayoutWithSearch/__snapshots__/index.spec.tsx.snap]: fixed at f4a6fb4f1c5a3f92abc8256fafb9511c28d0eab8

**Review by yee2542 — APPROVED** [review]

**Pull request merged**